Scripts/form.js

```diff
@@ -0,0 +1,44 @@
+// form.js
+
+import { createPost } from "./api.js";
+
+// Function to create a new post
+async function createNewPost() {
+  const titleInput = document.getElementById("new-post-title");
+  const bodyInput = document.getElementById("new-post-body");
+  const mediaInput = document.getElementById("new-post-media");
+
+  const title = titleInput ? titleInput.value : "";
+  const body = bodyInput ? bodyInput.value : "";
+  const media = mediaInput ? mediaInput.value : "";
+
+  try {
+    const newPost = await createPost(title, body, media);
+    console.log("New post created:", newPost);
```

Comment:
Display a message to the user in the html if the post is created successfully. Use a function similar to displayErrorMessage.